packages/bpk-component-rating/src/BpkRating.scss

```diff
@@ -0,0 +1,49 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016-2019 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import '~bpk-mixins';
+
+.bpk-rating {
+  display: grid;
+  grid-template-rows: auto;
+  grid-template-columns: max-content auto;
+  grid-auto-flow: column;
+  grid-column-gap: $bpk-spacing-sm;
+
+  &--base-rating {
+    width: $bpk-text-base-line-height * 2;
+    height: $bpk-text-base-line-height * 2;
+    padding: $bpk-spacing-sm;
+    grid-column: 1;
+    grid-row: 1 / 3;
+    border-radius: $bpk-text-xl-line-height;
```

Comment:
There should be a `border-radius-pill` token we can use here. If not, we can add one ourselves. I know RN has one but maybe it doesn't exist for web. What border radius are buttons using? Those are rounded so we should be able to mirror whatever's done there.

Reply:
So I tried with radius-pill and it produced the following output. I had originally hardcoded with a specific value and then found this one was close enough. So in this case I will create new tokens specific to this component 😄 

![Screenshot 2019-08-15 at 10 18 19](https://user-images.githubusercontent.com/8831547/63085736-1c9a4900-bf46-11e9-914d-9aae38159a5b.png)

Reply:
Ok cool, sounds good!

Reply:
Created new token to handle this